apps.yaml

```diff
@@ -396,3 +396,20 @@ EVA:
     categories:
         - data-visualization
         - data-analysis
+NanoBattVR:
+    metadata:
+        title: NanoBattVR
+        description: |
+            A virtual reality lab for exploring battery materials at the atomic level.
+            Simulate using either fast classical methods or deploy DFT calculation on a cluster.
+        authors: Z. Mahmood, I. Castelli, S. Lysgaard, M. Brandbyge, P. Bøggild
+        external_url: https://drive.google.com/drive/folders/1B3yrpWTZCHrKotWMuzhscNiD4PG_MYC7?usp=drive_link
+        documentation_url: https://github.com/NanoscaleSimulations/NanoBattVR_documentation
+        logo: https://nanoscalesimulations.com/Logo/Favicon/Favicon%20(M%C3%B8rk%20baggrund).svg
+        state: development
+        version: '1.0'
+    categories:
+        - Virtual Reality
```

Comment:
if you use a new category i.e. a category that is not already a keyword in the `categories.yaml` file, then you need to add that category in the [category.yaml](https://github.com/big-map/big-map-registry/blob/main/categories.yaml) file along with a brief description of what that category means.